src/reanimated2/Colors.ts

```diff
@@ -559,15 +604,15 @@ export function processColorInitially(color) {
   return normalizedColor;
 }
 
-export function isColor(value) {
+export function isColor(value: unknown): boolean {
   'worklet';
   if (typeof value !== 'string') {
     return false;
   }
   return processColorInitially(value) != null;
 }
 
-export function processColor(color) {
+export function processColor(color: unknown): number | null | undefined {
```

Comment:
Why color is as `:unknown` type?

Reply:
It uses `processColorInitially` which uses `normalizeColor`, both of them can accept arbitrary arguments (they're returning nulls when they cannot process the arguments). I think typings should reflect that.

Right now, we type this function as:
`export function processColor(color: number | string): number;`
so we can also simplify those methods by using `color: number | string`, casting it internally to unknown (to support JS users too), and e.g. throw when input is invalid. What do you think?